docs/CRA-Brief-Guide-for-OSS-Developers.md

```diff
@@ -0,0 +1,47 @@
+4. Requiring personal data for reasons other than for very narrow reasons (improving the security, compatibility, or interoperability of the software)
+5. Accepting donations exceeding costs.
+
+If you’re putting a PDE on the market in the course of a commercial activity, then you’re considered a [manufacturer](https://eur-lex.europa.eu/eli/reg/2024/2847/oj#art_3) and the CRA applies. This is true even if the PDE is open source software. That’s not a disaster. The CRA has a set of requirements that take work to achieve, but the requirements are intended to be *achievable*.
+
+### If your Open Source Software (OSS) is used by a manufacturer
+
+Manufacturers may integrate OSS components into their product that is put on the market. The CRA ***does*** apply to manufacturers, because they are putting PDEs on the market with commercial intent. The manufacturer is responsible for all parts of the product, including components from third parties. The manufacturer must [perform “due diligence”](https://eur-lex.europa.eu/eli/reg/2024/2847/oj#art_13) to determine what components to use, and must also [report component vulnerabilities](https://eur-lex.europa.eu/eli/reg/2024/2847/oj#art_13) to the component maintainer and upstream fixes if they have any. Using an OSS component in a product makes the manufacturer responsible for its use. As a result, it’s expected that some OSS will be more thoroughly assessed, and it’s likely that there will be a preference for more secure OSS. Manufacturers may sometimes [change how they interact with non-commercial OSS](https://eviltux.com/2025/04/25/what-open-source-developers-need-to-know-about-the-eu-cyber-resilience-act-cra/) due to the CRA. So even developers not directly subject to the CRA should learn more about the CRA and work to create more secure software.  These Manufacturer requirements may generate more interest in your software and your practices, which may spawn requests to the project for documentation, patches, or other artifacts such as a Software Bill of Materials (SBOM).
+
+Organizations that [systematically provide sustained support for developing one or more OSS projects intended for commercial activities](https://eur-lex.europa.eu/eli/reg/2024/2847/oj#art_3), but don’t fill another role like “manufacturer”, may be considered “Open Source Software Stewards” under the CRA. Stewards have fewer obligations than manufacturers, but they have a few [obligations](https://eur-lex.europa.eu/eli/reg/2024/2847/oj#art_24) such as providing a coordinated vulnerability disclosure (CVD) policy, cooperating with market surveillance at their request, providing certain kinds of documentation, reporting known actively exploited vulnerabilities, notifying about severe incidents, informing impacted users, and providing mitigation. There is no requirement for an OSS project to have a steward. However, an OSS project may *choose* to be supported by a steward (who must then meet its obligations).
```

Comment:
> but don’t fill another role like “manufacturer”

.. for a particular instance making the software available.

In other words, it may well be possible that you are both a manufacturer and a steward for the same tarball, just with respect to different audiences. See also https://github.com/orcwg/cra-hub/pull/226#issuecomment-2891588708